tools/wasm2c/base.c

```diff
@@ -10,10 +10,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/stat.h>
-#include <sys/types.h>
 #include <time.h>
-#include <unistd.h>
+
+// ssize_t detection: usually stdint provides it, but not on windows apparently
+#ifndef ssize_t
+#ifdef _WIN64
```

Comment:
`if __SIZE_WIDTH__ == 64`  ?   (__SIZE_WIDTH__ is a clang pre-defined macro)

Reply:
From some searching, it doesn't seem to exist in MSVC?

Reply:
But we are using clang aren't we?  Not MSVC?   I'm confused now..

Reply:
Just on the bots, but we do want this code to work on either clang or msvc on windows, i think.

Reply:
Ah... right.. in that case we don't want native_clang at all really... we want the system compiler I guess, which is currently a different thing.